README.md

```diff
@@ -2,13 +2,12 @@
 
 # Awesome Books Project
 
-> A web app to add your books in a digital shelf. This webapp is built with vanila javascript. A local storage option is added with this project, so that a user can store data in this is web app. 
+> A web app to add your books in a digital shelf. This webapp is built with vanila javascript. A local storage option is added with this project, so that a user can store data in this is web app.
```

Comment:
- [ ] The screenshots are not showing in the readme file. Please kindly fix this issue 📸

Reply:
Alright, well noted. I am going to update that